Resources/Locale/en-US/game-ticking/game-presets/preset-thief.ftl

```diff
@@ -0,0 +1,8 @@
+thief-role-greeting = 
+    You are a criminal. 
+    A thief looking to add to his collection, 
+    some property of Nanotrasen station. 
+    Choose your starting equipment, 
+    and try to do your job discreetly. 
+    You are not a killer, 
+    and violence is not acceptable to you.
```

Comment:
```suggestion
```
...continued from the above post since I'm not sure how to get multiple different lines that aren't next to eachother in a review comment. This should be removed for redundancy due to what was suggested above + pacifism already accomplishes the goals of this so making it a "rule" isn't really needed, people circumventing their pacifism to do their objectives should probably be encouraged since it encourages creative approaches.